mmseg/models/backbones/fast_scnn.py

```diff
@@ -136,11 +141,13 @@ def __init__(self,
             conv_cfg=self.conv_cfg,
             norm_cfg=self.norm_cfg,
             act_cfg=self.act_cfg,
-            align_corners=align_corners)
+            align_corners=True)
```

Comment:
`align_corners=True` -->  `align_corners=align_corners`
We can also have an ablation study for `align_corners`.